### DISCUSSION TITLE
Seeking Feedback for Receevi: Let's Make it Even Better! #2

### DISCUSSION THREAD

**Discussion by junedkhatri31** [receevi/receevi#3] [Ideas]
Hello!

We're excited to share our open-source "Receevi" with the community. We've created a basic app that displays messages sent by users, and now we're eager to hear from you! What kind of features would you like to see in the future releases of this project? We would also love to know why you're here in this Github repository. Your input and feedback will help us make this project even better. Let us know your thoughts in the comments section below.

Thank you for your support!

**Comment by hssan97** [comment]
Hi Dear
thanks for your project 
i had a small bug messages not updating until i refresh the page 
can you make it auto ?

**Reply by junedkhatri31** [reply]
Thanks for trying out the project. Will check it out. Can you please create an issue?

**Reply by junedkhatri31** [reply]
Hi @hssan97, This bug is now fixed. Thanks

**Comment by maliksoban** [comment]
**Hello @junedkhatri31, **
we appreciate your efforts and its a great work what we want from you guys is that supabase is used in this project what you think which package suits for this one and also can we use other databases like MongoDB .?
Also for the future releases we want that make some automation bot to help businesses with some predefined messages to get trigger on some specific keywords. 
Also a contact list to be store the name of the user who is sending the message.
Further we are happy to contribute in this repository as a team to bring more better product if you want just leave an mail and we will co ordinate there sobankh4@gmail.com

**Comment by Noddesk** [comment]
its good to create a you tube video regarding installation. Try rerunning the command with --debug to troubleshoot the error.
ebin@ebin-desktop:~/Documents/receevi/receevi-main$ supabase functions deploy
Bundling Function: sync-message-templates
Download https://deno.land/std@0.168.0/http/server.ts
Download https://esm.sh/@supabase/supabase-js@2
Download https://esm.sh/v133/@supabase/node-fetch@2.6.14/denonext/node-fetch.mjs
Download https://deno.land/std@0.168.0/async/mod.ts
Download https://esm.sh/@supabase/supabase-js@2.45.4
Download https://deno.land/std@0.168.0/async/abortable.ts
Download https://deno.land/std@0.168.0/async/deadline.ts
Download https://deno.land/std@0.168.0/async/debounce.ts
Download https://deno.land/std@0.168.0/async/deferred.ts
Download https://deno.land/std@0.168.0/async/delay.ts
Download https://deno.land/std@0.168.0/async/mux_async_iterator.ts
Download https://deno.land/std@0.168.0/async/pool.ts
Download https://deno.land/std@0.168.0/async/tee.ts
Download https://deno.land/std@0.168.0/async/retry.ts
Download https://esm.sh/v135/@supabase/auth-js@2.65.0/esnext/auth-js.mjs
Download https://esm.sh/v135/@supabase/functions-js@2.4.1/esnext/functions-js.mjs
Download https://esm.sh/v135/@supabase/realtime-js@2.10.2/esnext/realtime-js.mjs
Download https://esm.sh/v135/@supabase/postgrest-js@1.16.1/esnext/postgrest-js.mjs
Download https://esm.sh/v135/@supabase/storage-js@2.7.0/esnext/storage-js.mjs
Download https://esm.sh/v135/@supabase/node-fetch@2.6.15/esnext/node-fetch.mjs
Download https://esm.sh/v135/@supabase/supabase-js@2.45.4/esnext/supabase-js.mjs
Download https://esm.sh/v135/buffer@6.0.3/esnext/buffer.bundle.mjs
Download https://esm.sh/v135/ws@8.17.1/esnext/ws.mjs
Error: failed to create the graph

Caused by:
    Module not found "file:///home/ebin/Documents/receevi/receevi-main/supabase/functions/_shared/database.types.ts".
        at file:///home/ebin/Documents/receevi/receevi-main/supabase/functions/_shared/supabase_types.ts:2:26
error running container: exit 1

getting this error on ubuntu

**Reply by junedkhatri31** [reply]
Currently debugging the same issue. Thanks

**Comment by HighLiuk** [comment]
@junedkhatri31 Hello.
Not sure what this project is capable of. Can I send bulk WA campaigns from the UI?

**Reply by junedkhatri31** [reply]
Yes, you can. Do you have variables in your template which you want to bulk send?

**Reply by HighLiuk** [reply]
Sure. Can I have variables both on body, header and even buttons? Since WA API allows for them

**Reply by junedkhatri31** [reply]
Currently adding support for variables is in progress. However you can send bulk messages if your template doesn't have variables.

**Reply by HighLiuk** [reply]
Well nice enough ;) now I know what this project is about and that is actively maintained

**Reply by junedkhatri31** [reply]
I would like to understand your requirements. Can we connect?

**Reply by junedkhatri31** [reply]
Thanks

**Comment by sharadagg** [comment]
Hi

Started testing it yesterday. Quite happy with where this project is already

Things that work really well.
1. Chat interface is very quick. It updates instantly on receiving any incoming message
2. Able to add contacts from the web interface
3. Able to send messages on active chats

Features that will be super helpful
1. An api to add contacts from other system
2. A user friendly way to send template messges to contacts which are already added
3. Make the interface moble responsive so people can use it on the go
4. While sending template mesages - currenlty in the UI Whatsapp formatting doesn't show properly - though the message gets delivered in correct format.
5. While adding agents - somehow the confirmation emails have a redirect to localhost:3000 instead of the online supabase (maybe I have messed up something here)

**Comment by nikolaz111** [comment]
Hi,

I want to start using the project to be able to interact easily with the people responding to my template initiated messages. I'm wondering if the project is still under active development. When I finish testing the project, I'll bring my feedback.

**Reply by junedkhatri31** [reply]
The project is not being maintained.

**Comment by JimPresting** [comment]
@junedkhatri31 is there still active development going on?

**Reply by junedkhatri31** [reply]
Not right now.

**Comment by rebornleads-ai** [comment]
@junedkhatri31 , I guess this has been abandoned? Just stumbled into this trying to find a way to visualize messages sent/received via WA Business API. I can't believe WA app does not support that itself.

**Reply by junedkhatri31** [reply]
I'm sorry to say but I am not currently maintaining this. Will be back to this project one day.

**Comment by naveennahata** [comment]
Hi is it working. I have done the setup seeing webhook perfectly fine database entry of messages are there. However not seeing messages in ui.
<img width="1334" height="512" alt="Screenshot 2025-12-11 at 12 21 58 AM" src="https://github.com/user-attachments/assets/c1b74f44-dac6-4852-8f88-45c0fe05e72a" />

**Reply by naveennahata** [reply]
@junedkhatri31 Appreciate for any suggestion on this.
